### DISCUSSION TITLE
Exercise/HW 2 Q#6

### DISCUSSION THREAD

**Discussion by emilyyli** [LiNk-NY/IntroR#37] [General]
<img width="852" alt="image" src="https://github.com/CUNYSPHCode/IntroR/assets/171090659/3830b423-f516-4429-9769-97bd84b9aa16">

@ralythompson24 Out of curiosity, if we don't include the code byrow as part of our answer, does the matrix get automatically filled out by column or row? I know if we set the byrow= FALSE (the default), it would fill it out by column. Additionally, would this be relevant to the question here or more of a personal preference to include it in our answer?

**Comment by ralythompson24** [comment]
Hi @emilyyli, great question. So by default, the function assumes the argument `byrow=FALSE`, meaning that it will fill out the matrix by columns (so starting with top left, going down the first column). If we don't specify anything for `byrow` this default behavior applies. If we wanted to fill out the matrix by rows (going left to right starting with the first cell), we would have to specify `byrow=TRUE`. 

I didn't ask for you to do it any particular way in this example so you could leave `byrow` as-is or specify it yourself. This problem doesn't have any context to it, but there may be circumstances where you'd want the matrix values to order left to right vs. top to bottom. Depends on what you're trying to do with your matrix.

**Reply by emilyyli** [reply]
That makes sense, thank you!
